packages/manager/apps/web/client/app/domain/zone/record/DNAME/domain-zone-record-DNAME.html

```diff
@@ -34,26 +34,6 @@
                     ></span>
                 </span>
             </div>
-            <small
-                class="help-block"
-                data-ng-if="ctrl.targetIsRelativeDomain()"
```

Comment:
is ctrl.targetIsRelativeDomain() still in use?
Same question for the key "domain_configuration_dns_entry_relative_warning_title" and "domain_configuration_dns_entry_relative_warning_tooltip".

Reply:
yes, these functions and translations are used in other templates like MX and SRV

Reply:
i removed these usages too